lomap/graphgen.py

```diff
@@ -755,8 +790,32 @@ def max_dist_mol(mol):
         os.system(cmd)
         os.remove(self.dbase.options.name+'_tmp.dot')
         shutil.rmtree(directory_name, ignore_errors=True)
-
-        
+    #sliu 07/27 add function to output the score and connectivity txt file
```

Comment:
dated changelog not necessary -- though comments explaining what is being done here are always good.